src/App.tsx

```diff
@@ -540,222 +540,254 @@ function App(): ReactElement {
+
+          <LabeledDropdown
             disabled={disableUi}
+            label="Dataset"
+            selected={datasetName}
+            buttonType="primary"
+            items={urlUtils.getDatasetNames(datasetName, collectionData || null)}
             onChange={handleDatasetChange}
-            value={datasetName}
-          >
-            {urlUtils.getDatasetNames(datasetName, collectionData || null).map((name) => {
-              return (
-                <option value={name} key={name}>
-                  {name}
-                </option>
-              );
-            })}
-          </select>
-        </label>
-        <label>
-          Feature
-          <select name="Feature" disabled={disableUi} onChange={handleFeatureChange} value={featureName}>
-            {dataset?.featureNames.map((name) => {
-              return (
-                <option value={name} key={name}>
-                  {name}
-                </option>
-              );
-            })}
-          </select>
-        </label>
-
-        {/** Color Ramp */}
-        <div className={styles.labeledColorRamp}>
-          <input
-            type="number"
-            style={{ width: "50px", textAlign: "start" }}
-            value={colorRampMin}
-            onChange={(event) => {
-              setColorRampMin(event.target.valueAsNumber);
-            }}
-            min="0"
           />
+          <LabeledDropdown
+            disabled={disableUi}
+            label="Feature"
+            selected={featureName}
+            items={dataset?.featureNames || []}
+            onChange={handleFeatureChange}
+          />
+
           <div className={styles.colorRampContainer}>
             <span
               ref={colorRampRef}
               className={`${styles.colorRamp} ${disableUi ? styles.disabled : ""}`}
               onClick={(event) => handleColorRampClick(event)}
             ></span>
           </div>
-          <input
-            type="number"
-            style={{ width: "80px", textAlign: "start" }}
-            value={colorRampMax}
-            onChange={(event) => {
-              setColorRampMax(event.target.valueAsNumber);
-            }}
-            min="0"
-          />
-          <button onClick={handleResetRangeClick}>Reset range</button>
-          <label>
-            <input
-              type="checkbox"
-              checked={isColorRampRangeLocked}
-              onChange={() => {
-                // Invert lock on range
-                setIsColorRampRangeLocked(!isColorRampRangeLocked);
+        </div>
+        <div className={styles.headerRight}>
+          <Button type="link" className={styles.copyUrlButton} onClick={openCopyNotification}>
+            <LinkOutlined />
+            Copy URL
+          </Button>
+          <Button type="primary">Export</Button>
+          <Button type="primary">Load</Button>
+        </div>
+      </div>
+
+      <div className={styles.viewport}>
```

Comment:
Is viewport the right name for this? It's all the content underneath the header.

Reply:
I agree there could be some better name, so that viewport is really just the view of the cell image.  And in the future we might have more viewports.   When I think of a header, I think of the stuff below it as the main content area....?

Reply:
That makes sense, I'll try renaming it to `content` for now.